server/core/src/main/java/dev/slimevr/osc/VMCHandler.java

```diff
@@ -453,31 +452,37 @@ && isRightArmUnityBone(bone))
 					}
 				}
 
-				for (Tracker shareableTracker : computedTrackers) {
-					oscArgs.clear();
-					oscArgs.add(String.valueOf(shareableTracker.getId()));
-					addTransformToArgs(
-						shareableTracker.getPosition(),
-						shareableTracker.getRotation()
-					);
-					String address;
-					TrackerPosition role = shareableTracker.getTrackerPosition();
-					if (role == TrackerPosition.HEAD) {
-						address = "/VMC/Ext/Hmd/Pos";
-					} else if (
-						role == TrackerPosition.LEFT_HAND || role == TrackerPosition.RIGHT_HAND
-					) {
-						address = "/VMC/Ext/Con/Pos";
-					} else {
-						address = "/VMC/Ext/Tra/Pos";
-					}
-					oscBundle
-						.addPacket(
-							new OSCMessage(
-								address,
-								oscArgs.clone()
-							)
+				for (Tracker tracker : computedTrackers) {
+					if (tracker.getStatus() != TrackerStatus.DISCONNECTED) {
```

Comment:
Should the status also be checked for `TrackerStatus.ERROR` here?

Reply:
sure